src/panels/lovelace/cards/hui-horizontal-stack-card.ts

```diff
@@ -29,7 +29,7 @@ class HuiHorizontalStackCard extends HuiStackCard {
         }
         #root > * {
           flex: 1 1 0;
-          margin: 0 4px;
+          margin: var(--stack-margin, var(--horizontal-stack-margin, 0 4px));
```

Comment:
This should be the other way? horizontal should override general stack?

Also name it `stack-card-margin`